js/works.js

```diff
@@ -0,0 +1,80 @@
+const projectsInfo = [
+  {
+    title: 'Facebook 360',
+    desc: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    langs: ['Html', 'Css', 'Javascript'],
+    imgCard: '.././assets/images/Portfolio1.png"',
+    cannopy: 'FACEBOOK',
+    job: 'Full Stack Dev',
+    live: '#',
+    source: '#',
+  },
+  {
+    title: 'Multi-Post Stories',
+    desc: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    langs: ['HTML', 'Ruby on rails', 'Javascript', 'CSS'],
+    imgCard: '.././assets/images/Portfolio3.png',
+    cannopy: 'FACEBOOK',
+    job: 'Full Stack Dev',
+    live: '#',
+    source: '#',
+  },
+  {
+    title: 'Tonic',
+    desc: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    langs: ['HTML', 'CSS', 'Javascript'],
+    imgCard: '.././assets/images/Portfolio2.png',
+    cannopy: 'CANOPY',
+    job: 'Back End Dev',
+    live: '#',
+    source: '#',
+  },
+  {
+    title: 'Uber Navigation',
+    desc: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    langs: ['HTML', 'Ruby on rails', 'Javascript', 'CSS'],
+    imgCard: '.././assets/images/Portfolio4.png',
+    cannopy: 'Uber',
+    job: 'Lead Developer',
+    live: '#',
+    source: '#',
+  },
+];
+
+const projects = document.querySelector('.works');
```

Comment:
- [ ] The popup image is stretching, kindly make sure the image is displayed properly so your website looks exactly as the provided Figma design

![Screenshot_20220907_215907](https://user-images.githubusercontent.com/25227022/188978417-59d5dd22-5dda-4c0e-95fe-5664515d1330.png)

Reply:
Hello @Ismailco 
thanks about the review, How about this?

Reply:
@Mhamad-Raad, you are good to go, I see that you got approved, keep up the good work 💯